global-api/routes/catalogue_last_update_endpoint.py

```diff
@@ -14,9 +14,9 @@ def db_query():
             FROM datasource;
             """
         )
-
         result = session.execute(query).fetchall()
-
+    if not result or result[0][0] is None:
+        return None
```

Comment:
Throwing a 404 error might make sense here? Or is that covered by returning None?